### PR TITLE
Chem rework mk3  the nerfening

### PR DESCRIPTION
**Pull request by jroinc**
## About The Pull Request

Chem rework mk3: The nerfening

## Why It's Good For The Game

Marine pre-med bad, fite me

## Changelog
:cl:
balance: Neurotoxin now might do enough stuff to be worth injecting.
balance: Larval toxin now purges meds at 3u/tick. 4 pills are no longer a guaranteed death as human
balance: Marine pre-med worsened even more to disincentives unga rushes.
balance: Oxycodone now has a drawback.
balance: CLB contents changed to support more "frontline" combat.
add: Recipe for perdiox+
remove: Medic vendor shit that's free or completely unused.

Full details in commits. 
/:cl:

### CODE REVIEW

**Comment by Pariah919** [comment]
lol

**Comment by jroinc** [comment]
Functioning as intended, no more additions to be made.

Highlights- If you double-dose on the 3 core meds now, you actually feel the pain of your wounds.

Neurotoxin's more badass and a fully channeled sting's something to be feared now. Pop antitox, pop synaptizine, pop hypervene, drink water, or run, cause otherwise you gonna be a slow boi, and then a stunned boi.

Oxycodone, and antitox in high doses gradually shut down running. Be cautious when pre-medding.

Medical belts might have a different niche than lifesaver belts instead of being arguably a worse lifesaver belt.

**Comment by XElectricX** [comment]
Shit nerfs. Medical belt is the best item for a medic/doctor with all the pills that you can cram into it. Injectors are one time use. Make the lifesaver the "frontline" if you want something in that role. I don't know anyone who uses lifesaver belts.

Also I use oxycodone, ree.

**Comment by jroinc** [comment]
> Shit nerfs. Medical belt is the best item for a medic/doctor with all the pills that you can cram into it. Injectors are one time use. Make the lifesaver the "frontline" if you want something in that role. I don't know anyone who uses lifesaver belts.
> Also I use oxycodone, ree.

Lifesaver belts are generally the "go-to" belt for a medic and are considered the better medical belt, as they hold nearly everything a medic belt does, with the advantage of holding an extra 7 items. Only real disadvantage lifesaver belts have are not holding rollerbeds and defibs, but I (personally) keep those as satchel items.

The specifics of the change are that you lost hypervene, dexalin, and spaceacillin (which aren't ever needed in pill-bottle amounts front-line) and perdiox and QC (which might be needed, but feel free to either grab a lifesaver bag or edit the loadout in medical with vendors which you have default access to roundstart).

For those 5 slots you lose, you get 11 combat injectors with 5u bicar, kelo, and trama in one-shot doses. They ARE re-stockable, either at medbay, or using a hacked wall vendor on the Alamo, but they aren't all intended to be restocked. The idea is you use the 2 in it, then pull a syringe case with 3 injectors out, then dump the injectors back into the belt, then toss the case. Awkward, but it doubles storage space so it gud.

**Comment by Pariah919** [comment]
Yeah I gotta disagree with the medical belt changes here.

**Comment by jroinc** [comment]
> Yeah I gotta disagree with the medical belt changes here.

What specific changes would you like to see?

**Comment by jroinc** [comment]
Medical belt change "reverted"

Lifesaver belt ~sacrificed~ changed in it's stead. Lifesaver belt is more auto-injector and rapid-care focused, while medical belt retains non-core pill bottles and medical machinery items.

**Comment by Pariah919** [comment]
Gotta say a big no to those lifesaver belt changes too.

**Comment by jroinc** [comment]
> Gotta say a big no to those lifesaver belt changes too, unless you readd quickclot and dex.

Not gonna. Use your injectors. Not having QC pill bottles is a legit weakness, but your 3-shot injector should cover most cases (unless dumbass just wants to keep patching it so he can stay on the field, in which case I'm not pandering to him). Dexalin can go die in a fire, use you dex+ injector.

Virgin dexalin pill- Has pill delay, heals ONE point of damage.

Chad dex+ injector- Heals all of it. Instantly.

Also, you can customize this shit, just grab a QC bottle and change it out if you really want them so bad. I'm sure I won't disincentivize being on QC constantly.

File: code/game/objects/items/reagent_containers/autoinjectors.dm [hunk 1]
```diff
@@ -61,9 +61,9 @@
 	name = "quick-clot autoinjector"
 	desc = "An autoinjector loaded with 10 units of quick-clot, a chemical designed to pause all bleeding. Renew doses as needed."
```
Comment (Rohesie) [inline code/game/objects/items/reagent_containers/autoinjectors.dm:62]:
```suggestion
	desc = "An autoinjector loaded with 30 units of quick-clot, a chemical designed to pause all bleeding. Renew doses as needed."
```

File: code/game/objects/items/reagent_containers/glass/bottle.dm [hunk 2]
```diff
@@ -224,6 +224,12 @@
 	icon_state = "bottle7"
 	list_reagents = list(/datum/reagent/toxin/xeno_neurotoxin = 30)
 
+/obj/item/reagent_containers/glass/bottle/xeno_growthtoxin
+	name = "\improper Growth toxin bottle"
+	desc = "A small bottle. Contains artificialy synthesized growth toxin- useful for researchers, or as a very risky medicine."
```
Comment (Rohesie) [inline code/game/objects/items/reagent_containers/glass/bottle.dm:229]:
```suggestion
	desc = "A small bottle. Contains artificialy synthesized growth toxin - useful for researchers, or as a very risky medicine."
```

File: code/game/objects/items/storage/firstaid.dm [hunk 3]
```diff
@@ -223,6 +223,17 @@
...
 
+/obj/item/storage/syringe_case/combat
+
+/obj/item/storage/syringe_case/combat/Initialize(mapload, ...)
+	. = ..()
+	new /obj/item/reagent_containers/hypospray/autoinjector/combat(src)
+	new /obj/item/reagent_containers/hypospray/autoinjector/combat(src)
+	new /obj/item/reagent_containers/hypospray/autoinjector/combat(src)
+
+
```
Comment (Rohesie) [inline code/game/objects/items/storage/firstaid.dm:234]:
```suggestion

/obj/item/storage/syringe_case/combat

/obj/item/storage/syringe_case/combat/PopulateContents()
	. = ..()
	new /obj/item/reagent_containers/hypospray/autoinjector/combat(src)
	new /obj/item/reagent_containers/hypospray/autoinjector/combat(src)
	new /obj/item/reagent_containers/hypospray/autoinjector/combat(src)
```

File: code/game/objects/items/reagent_containers/autoinjectors.dm (hunk 1 above)
Comment (jroinc) [reply code/game/objects/items/reagent_containers/autoinjectors.dm:62]: shit missed that, thanks

File: code/game/objects/items/storage/firstaid.dm (hunk 3 above)
Comment (jroinc) [reply code/game/objects/items/storage/firstaid.dm:234]: will do soon

**Comment by jroinc** [comment]
Assuming the requested change I did works, this has been tested enough that I'm satisfied it's a good addition.

**Comment by Rohesie** [comment]
Code looks fine.

**Pull request merged**